frontend/src/actions/authActions.ts

```diff
@@ -0,0 +1,50 @@
+import * as types from "./types";
+import AuthService from "../services/authService";
+
+export const register = (firstName, lastName, email, password) => (dispatch) => {
+    return AuthService.register(firstName, lastName, email, password).then(
```

Comment:
i think register should also dispatch with a payload with user data, so a user can register then gets automatically logged in. With the current setup, registering an account won't update the state with user info/access token